### PR TITLE
feat: CEA608/708 parser

### PR DESCRIPTION
**Pull request by hernanr99**
## Description
We've implemented support for the CEA608/708 subtitles standard to our library, expanding its capabilities. In this update, alongside the feature implementation, we've initiated the integration with a basic set of tests. 

## Requirements Checklist
- [X] Unit Tests updated or fixed
- [ ] Docs/guides updated

### CODE REVIEW

File: lib/src/captions/cea608-708/utilities/utils.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+export function createCmdHistory(): CmdHistory {
+	return {
+		a: null,
+		b: null,
+	};
+}
+
+
+export function extractCea608DataFromRange(raw: any, cea608Range: any) {
```
Comment (littlespex) [inline lib/src/captions/cea608-708/utilities/utils.ts:43]: Add proper data types for these args.

**Review by littlespex — CHANGES_REQUESTED** [review]

**Review by mpollingerQualabs — DISMISSED** [review]
LGTM

**Comment by avelad** [comment]
One question, do you support the subtitles described in https://github.com/videojs/mux.js/pull/439 ?

**Comment by mpollingerQualabs** [comment]
> One question, do you support the subtitles described in [videojs/mux.js#439](https://github.com/videojs/mux.js/pull/439) ?

Hey @avelad, do you have a playlists we could test with? Or should we try with the one on that PR?

**Comment by avelad** [comment]
> > One question, do you support the subtitles described in [videojs/mux.js#439](https://github.com/videojs/mux.js/pull/439) ?
> 
> Hey @avelad, do you have a playlists we could test with? Or should we try with the one on [that PR](https://solutions.brightcove.com/wseymour/708-testing/playlist.m3u8)?

Yes, you can use:
Example Test Playlist: https://solutions.brightcove.com/wseymour/708-testing/playlist.m3u8

**Comment by mpollingerQualabs** [comment]
> > > One question, do you support the subtitles described in [videojs/mux.js#439](https://github.com/videojs/mux.js/pull/439) ?
> > 
> > 
> > Hey @avelad, do you have a playlists we could test with? Or should we try with the one on [that PR](https://solutions.brightcove.com/wseymour/708-testing/playlist.m3u8)?
> 
> Yes, you can use: Example Test Playlist: https://solutions.brightcove.com/wseymour/708-testing/playlist.m3u8

We've tried with that one but it doesn't seem to have a subtitle playlist. Could you provide us another one? Or what specific type of subtitle should we test?

**Review by fyoungqualabs — APPROVED** [review]

**Review by mpollingerQualabs — APPROVED** [review]

**Pull request merged**